### PR TITLE
chore(dew): modify KMS key resource lint error and update document

### PR DESCRIPTION
**Pull request by ruwenqiang123**
**What this PR does / why we need it**:

Modify KMS key resource lint error and update document.

**Which issue this PR fixes**:
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
fixes #xxx

**Special notes for your reviewer**:

**Release note**:

```release-note

```

## PR Checklist

* [x] Tests added/passed.

```
$ make testacc TEST="./huaweicloud/services/acceptance/dew" TESTARGS="-run TestAccKmsKey_basic"
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./huaweicloud/services/acceptance/dew -v -run TestAccKmsKey_basic -timeout 360m -parallel 4
=== RUN   TestAccKmsKey_basic
--- PASS: TestAccKmsKey_basic (112.87s)
PASS
ok      github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/acceptance/dew       112.923s
```
```
$ make testacc TEST="./huaweicloud/services/acceptance/dew" TESTARGS="-run TestAccKmsKey_ExternalKey"
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./huaweicloud/services/acceptance/dew -v -run TestAccKmsKey_ExternalKey -timeout 360m -parallel 4
=== RUN   TestAccKmsKey_ExternalKey
--- PASS: TestAccKmsKey_ExternalKey (32.02s)
PASS
ok      github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/acceptance/dew       32.081s
```
```
$ make testacc TEST="./huaweicloud/services/acceptance/dew" TESTARGS="-run TestAccDataEncryptDecrypt_basic"
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./huaweicloud/services/acceptance/dew -v -run TestAccDataEncryptDecrypt_basic -timeout 360m -parallel 4
=== RUN   TestAccDataEncryptDecrypt_basic
=== PAUSE TestAccDataEncryptDecrypt_basic
=== CONT  TestAccDataEncryptDecrypt_basic
--- PASS: TestAccDataEncryptDecrypt_basic (70.45s)
PASS
ok      github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/acceptance/dew       70.544s
```
```
$ make testacc TEST="./huaweicloud/services/acceptance/dew" TESTARGS="-run TestAccDataSourceKmsKeys_basic"
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./huaweicloud/services/acceptance/dew -v -run TestAccDataSourceKmsKeys_basic -timeout 360m -parallel 4
=== RUN   TestAccDataSourceKmsKeys_basic
=== PAUSE TestAccDataSourceKmsKeys_basic
=== CONT  TestAccDataSourceKmsKeys_basic
--- PASS: TestAccDataSourceKmsKeys_basic (57.49s)
PASS
ok      github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/acceptance/dew       57.541s
```
```
$ make testacc TEST="./huaweicloud/services/acceptance/dew" TESTARGS="-run TestAccKmsKeyDataSource_basic"
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./huaweicloud/services/acceptance/dew -v -run TestAccKmsKeyDataSource_basic -timeout 360m -parallel 4
=== RUN   TestAccKmsKeyDataSource_basic
=== PAUSE TestAccKmsKeyDataSource_basic
=== CONT  TestAccKmsKeyDataSource_basic
--- PASS: TestAccKmsKeyDataSource_basic (41.91s)
PASS
ok      github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/acceptance/dew       42.034s
```
```
$ make testacc TEST="./huaweicloud/services/acceptance/dew" TESTARGS="-run TestAccKmsKeyDataSource_withTags"
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./huaweicloud/services/acceptance/dew -v -run TestAccKmsKeyDataSource_withTags -timeout 360m -parallel 4
=== RUN   TestAccKmsKeyDataSource_withTags
=== PAUSE TestAccKmsKeyDataSource_withTags
=== CONT  TestAccKmsKeyDataSource_withTags
--- PASS: TestAccKmsKeyDataSource_withTags (40.42s)
PASS
ok      github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/acceptance/dew       40.493s
```
```
$ make testacc TEST="./huaweicloud/services/acceptance/dew" TESTARGS="-run TestAccKmsKeyDataSource_withEpsId"
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./huaweicloud/services/acceptance/dew -v -run TestAccKmsKeyDataSource_withEpsId -timeout 360m -parallel 4
=== RUN   TestAccKmsKeyDataSource_withEpsId
=== PAUSE TestAccKmsKeyDataSource_withEpsId
=== CONT  TestAccKmsKeyDataSource_withEpsId
--- PASS: TestAccKmsKeyDataSource_withEpsId (44.37s)
PASS
ok      github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/acceptance/dew       44.416s
```

* [x] Documentation updated.
* [x] Schema updated.
* [ ] CheckDeleted.

  - **a. During query operation (Read Context)**
    aa. Resource not found
    \>>>>>> Paste the screenshot here <<<<<<

    
  - **b. During delete/disassociate/unbind operation (Delete Context)**
    ba. Resource not found
    \>>>>>> Paste the screenshot here <<<<<<

### CODE REVIEW

File: huaweicloud/services/dew/resource_huaweicloud_kms_key.go [hunk 1]
```diff
@@ -190,36 +189,39 @@ func ResourceKmsKeyCreate(ctx context.Context, d *schema.ResourceData, meta inte
...
+	_, err = stateConf.WaitForStateContext(ctx)
 	if err != nil {
-		return diag.Errorf("error waiting for KMS key (%s) to become ready: %s", v.KeyID, err)
+		return diag.Errorf("error waiting for KMS key (%s) to become ready: %s", d.Id(), err)
 	}
-	keyInfo := result.(*keys.Key)
-	if !d.Get("is_enabled").(bool) && keyInfo.KeyState == EnabledState {
-		key, err := keys.DisableKey(kmsKeyV1Client, v.KeyID).ExtractKeyInfo()
+
+	if !d.Get("is_enabled").(bool) && resp.KeyState == EnabledState {
```
Comment (deer-hang) [inline huaweicloud/services/dew/resource_huaweicloud_kms_key.go]: Don't change the original logic.

File: huaweicloud/services/dew/resource_huaweicloud_kms_key.go [hunk 2]
```diff
@@ -267,62 +268,62 @@ func ResourceKmsKeyCreate(ctx context.Context, d *schema.ResourceData, meta inte
...
-		d.Set("key_usage", v.KeyUsage),
-		d.Set("key_state", v.KeyState),
-		d.Set("keystore_id", v.KeyStoreID),
-		utils.SetResourceTagsToState(d, kmsKeyV1Client, "kms", d.Id()),
+		d.Set("key_id", key.KeyID),
+		d.Set("domain_id", key.DomainID),
+		d.Set("key_alias", key.KeyAlias),
+		d.Set("key_description", key.KeyDescription),
+		d.Set("key_algorithm", key.KeySpec),
+		d.Set("creation_date", utils.FormatTimeStampRFC3339(convertStrToInt(key.CreationDate)/1000, false)),
```
Comment (deer-hang) [inline huaweicloud/services/dew/resource_huaweicloud_kms_key.go]: This change is an incompatible change.

**Comment by ruwenqiang123** [comment]
```
$ make testacc TEST="./huaweicloud/services/acceptance/dew" TESTARGS="-run TestAccKmsKey_basic"
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./huaweicloud/services/acceptance/dew -v -run TestAccKmsKey_basic -timeout 360m -parallel 4
=== RUN   TestAccKmsKey_basic
--- PASS: TestAccKmsKey_basic (115.53s)
PASS
ok      github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/acceptance/dew       115.606s
```

**Pull request merged**